.github/workflows/codeql-analysis.yml

```diff
@@ -1,6 +1,10 @@
 name: CodeQL
 
-on: pull_request
+on:
+  pull_request:
```

Comment:
I think we need to remove this line

Reply:
Why? We still want this workflow to run on all PRs as well as pushes to master

Reply:
I think we need to delete `branch: master` and leave just `on: push`

Reply:
Oh right, I thought you were referring to the `pull_request` line!

Reply:
We've talked about this elsewhere and decided to leave it like it is for now